### PR TITLE
feat: homepage refresh (deprecated 🚨)

### PR DESCRIPTION
**Pull request by shamimbinnur**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
I know this is still a WIP, but want to point out that the images in the preview includes the older images. 

Screen Shot 2023-04-24 at 12 45 19 PM
Screen Shot 2023-04-24 at 12 45 13 PM

cc @getaheaddev

**Comment by bdougie** [comment]
We need a mobile view of this element.

 image

It's too small

**Comment by chhristopher** [comment]
Hey @shamimbinnur, communicating here for visibility. Per our discussion: Hi-res images and updated layout/copy. I'll wait for your green light to go through both the desktop and mobile versions and consolidate my feedback to give to you all at once.  Thanks!

File: components/sections/home-page/Hero.tsx [hunk 1]
```diff
@@ -34,10 +38,13 @@ const Hero = ({ data }: HeroProps) => {
               </Button>
             ))}
           </div>
+          <a className="text-sm text-wh pt-7" href="#">
+            Working with a team? Click here 
+          </a>
```
Comment (bdougie) [inline components/sections/home-page/Hero.tsx:43]: This should be a link to the previous landing page at the `/teams` route.

File: components/sections/home-page/Newsletter.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          </div>
+
+          <div className="absolute bottom-[32rem] left-0  largeTablet:hidden">
+            <Image alt="Doodles" src={StrokeMobile3} />
+          </div>
+        <div className="flex flex-col items-center gap-y-8">
+          
+        <Heading alignSmall="center">
+          $orange-to-yellowThe fast track$orange-to-yellow to open source
+        </Heading>
```
Comment (bdougie) [inline components/sections/home-page/Newsletter.tsx:35]: There are two newsletter sections and should be one. We only need email, no name as well.

Screen Shot 2023-04-26 at 5 33 13 AM

Screen Shot 2023-04-26 at 5 32 54 AM

**Review by bdougie — CHANGES_REQUESTED** [review]
1. Only one newsletter needed. Double checking with design see how that got missed
2. "Click here" needs to go to the previous landing page.

File: components/sections/home-page/Hero.tsx (hunk 1 above)
Comment (shamimbinnur) [reply components/sections/home-page/Hero.tsx:43]: Do you mean https://opensauced.pizza/teams?

Comment (bdougie) [reply components/sections/home-page/Hero.tsx:43]: > Do you mean https://opensauced.pizza/teams?

Correct

File: components/sections/home-page/Newsletter.tsx (hunk 2 above)
Comment (chhristopher) [reply components/sections/home-page/Newsletter.tsx:33]: Hey @shamimbinnur, let's delete the "Subscribe for Extra Sauce" newsletter section and keep the "The fast track..." section as is (without the 'Your Name' of course).  Could we also switch out the copy from "Sign Up" to "Subscribe"? 

Thanks

**Comment by chhristopher** [comment]
Screen Shot 2023-04-26 at 12 45 24 PM

Can we have the bottom copy "Working with the team? Click here" stylized like the other text? Let's have "Click here" highlighted in our gradient color . Also when we click on "Click here" it goes to a 404, we need to have it route to the enterprise site. @bdougie , did we want to have that opensauced.pizza/enterprise ?

**Comment by chhristopher** [comment]
The mobile site still have low-res photos and need to be retina scaled.  examples attached 


File (2)
File (1)
File

**Comment by shamimbinnur** [comment]
> The mobile site still has low-res photos and need to be retina scaled. examples attached
> 
> File (2) File (1) File

I have just realized that this issue is occurring on the Safari browser. I will explore ways to resolve it."

**Comment by chhristopher** [comment]
Thanks @shamimbinnur . We've also updated the repo segment on the mobile version, could you take a look and let me know if you have any questions?

**Pull request closed without merging**